### PR TITLE
correction for issue124

### PR DESCRIPTION
**Pull request by wdaniau**
Correction for issue 124 concerning multiple values in fortran.includePaths.

### CODE REVIEW

File: src/lib/helper.ts [hunk 1]
```diff
@@ -87,10 +87,11 @@ export const _loadDocString = (keyword: string) => {
 };
 
 export const getIncludeParams = (paths: string[]) => {
-  if (paths.length === 0) {
-    return '';
-  }
-  return '-I ' + paths.join(' ');
+  let arrayPaths: string[] = [];  
+  paths.forEach(function(element) {
+    arrayPaths.push("-I"+element);
```
Comment (krvajal) [inline src/lib/helper.ts]: I would rewrite this in a more declarative way. For example,
```
return paths.map(path => `-I ${path}`).join(" ")

```

File: src/features/linter-provider.ts [hunk 2]
```diff
@@ -107,7 +107,7 @@ export default class FortranLintingProvider {
     );
     let argList = [
       ...args,
-      getIncludeParams(includePaths), // include paths
+      ...(getIncludeParams(includePaths)), // include paths
```
Comment (krvajal) [inline src/features/linter-provider.ts]:
```suggestion
      ...getIncludeParams(includePaths), // include paths
```
No need for the extra parentesis.

File: src/lib/helper.ts (hunk 1 above)
Comment (wdaniau) [reply src/lib/helper.ts]: Done.
```
return paths.map(path => `-I${path}`)
```
By the way no join as the returned value is an array an expected to be an array in linter-provider.ts

File: src/features/linter-provider.ts (hunk 2 above)
Comment (wdaniau) [reply src/features/linter-provider.ts]: removed.

**Pull request merged**